homeassistant/components/matter/select.py

```diff
@@ -435,4 +435,65 @@ def _update_from_device(self) -> None:
         # don't discover this entry if the supported rinses list is empty
         secondary_value_is_not=[],
     ),
+    MatterDiscoverySchema(
+        platform=Platform.SELECT,
+        entity_description=MatterSelectEntityDescription(
+            key="LocalConfigDisabled",
+            entity_category=EntityCategory.CONFIG,
+            translation_key="local_config_disabled",
+            options=["Enable Local Configuration", "Disable Local Configuration"],
+            measurement_to_ha={
+                False: "Enable Local Configuration",
+                True: "Disable Local Configuration",
+            }.get,
+            ha_to_native_value={
+                "Enable Local Configuration": False,
+                "Disable Local Configuration": True,
```

Comment:
same comment as above

Reply:
Removed these lines. Will add back in a separate PR converting these to switch entities.